app/seller.html

```diff
@@ -105,122 +105,131 @@
-      field="email.name.raw" value="[[sellerDisplay.seller.emailAddress]]">
+    <!--
+      build the query for offer locations 
+    -->
+     <query-builder 
+      query="{{offerLocationQuery}}" 
+      query-template="{{config.queryTemplates.offerLocation}}"
+      field="seller.uri" value="[[sellerDisplay.seller._id]]">
     </query-builder>
 
     <elastic-search 
       client="[[esclient]]"
       index="[[config.elasticIndex]]"
-      query='[[sellerEmailAggs]]'
-      results="{{relatedEmails}}"
-      elastic-type='["seller"]'
+      query='[[offerLocationQuery]]'
+      results="{{offerLocationAggs}}"
+      elastic-type='["offer"]'
       lastError="{{error}}">
     </elastic-search>
 
     <json-transform
-     data-in="[[relatedEmails]]"
-     data-out="{{sellerDisplay.relatedEmails}}"
-     transform-function="[[clientConfig.transforms.seller.relatedEmails]]"
+     data-in="[[offerLocationAggs]]"
+     data-out="{{sellerDisplay.offerLocation}}"
+     transform-function="[[clientConfig.transforms.seller.offerLocationData]]"
      ></json-transform>
 
-    <!-- Related offers and people (full list) -->
-    <bool-query-builder
-      query="{{relatedEntityQuery}}"
-      query-template="{{config.queryTemplates.sellerPeopleAndOffers.query}}"
-      paths-to-values="{{config.queryTemplates.sellerPeopleAndOffers.pathsToValues}}"
-      value="[[sellerDisplay.seller._id]]">
-    </bool-query-builder>
+    <!--
+      Get all the offers for this phone
+     -->
+    <query-builder 
+      query="{{sellerOffersQuery}}" 
+      query-template="{{config.queryTemplates.commonMatchQueryOfferSorted}}"
+      field="seller.uri" value="[[sellerDisplay.seller._id]]">
+    </query-builder>
 
     <elastic-search 
       client="[[esclient]]"
       index="[[config.elasticIndex]]"
-      query='[[relatedEntityQuery]]'
-      page="{{pageNum}}"
+      query='[[sellerOffersQuery]]'
+      results="{{sellerOffers}}"
+      elastic-type='["offer"]'
+      loading="{{offersLoading}}"
       page-size="{{pageSize}}"
-      results="{{relatedResults}}"
-      elastic-type='["offer", "adultservice"]'
-      lastError="{{error}}"
-      loading="{{relatedEntitiesLoading}}">
+      page="{{pageNum}}"
+      lastError="{{error}}">
+
     </elastic-search>
 
     <json-transform
-     data-in="[[relatedResults]]"
-     data-out="{{relatedRecords}}"
-     transform-function="[[clientConfig.transforms.relatedEntities.combinedResults]]"
+     data-in="[[sellerOffers]]"
+     data-out="{{sellerDisplay.sellerOffers}}"
+     transform-function="[[clientConfig.transforms.seller.sellerOffersData]]"
      ></json-transform>
 
-    <!-- Related people (aggs) -->
+
+    <!-- Itenary component (aggs) -->
+
     <query-builder 
-      query="{{relatedServiceAggQuery}}" 
-      query-template="{{config.queryTemplates.sellerPeopleAggs}}" 
-      field="offers.seller.uri" value="[[sellerDisplay.seller._id]]">
+      query="{{iternararyAggQuery}}" 
```

Comment:
Is this supposed to be "itineraryAggQuery"?  If so, please change here and on line 221.